cmd/scheduler/app/options/options.go

```diff
@@ -87,6 +92,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	}
 
 	fs.BoolVar(&o.LeaderElection.LeaderElect, "leader-elect", true, "Enable leader election, which must be true when running multi instances.")
+	fs.StringVar(&o.LeaderElection.ResourceName, "leader-elect-resource-name", scheduler.DefaultScheduler, "The name of resource object that is used for locking during leader election.")
```

Comment:
I think we don't need this parameter, just use the scheduler name as the leader-elect resource name.

Reply:
+1

Reply:
In K8s, leader-elect-resource-name and schedulerName are not directly related. For example, kube-scheduler's resource name is 'kube-scheduler', schedulerName is 'default-scheduler'.

Reply:
I don't understand why we need to default the election resource name to schedulerName, I think it should be `karmada-scheduler`. 
If people deploy another scheduler along with `karmada-scheduler`, this flag should be specified.

By the way, this should be present in the documents, please track it in the umbrella issue.